homeassistant/components/sonos/media_player.py

```diff
@@ -474,20 +477,17 @@ def _play_favorite(self, favorite: DidlFavorite) -> None:
             soco.add_to_queue(favorite.reference)
             soco.play_from_queue(0)
 
-    @property  # type: ignore[misc]
-    def source_list(self) -> list[str]:
+    @property
+    def source_list(self) -> list[str | None]:
```

Comment:
You should never return `None` as an item in the list.

Reply:
It doesn't, this was just mistyped.